firebase-debug.1.log

```diff
@@ -0,0 +1,47 @@
+[debug] [2021-04-06T03:53:14.012Z] ----------------------------------------------------------------------
```

Comment:
I think this could probably be deleted? I don't think it needs to be in the project.

Reply:
or maybe added to the .gitignore? which do you think would be better?

Reply:
I'm not 100% sure, but I'm 99.9% sure you can just delete it. It looks like a file that got automatically. created when something went wrong. Safe to delete! 👍

Reply:
That's so weird... I totally agree with you - don't think we need it, but on opening my feature branch of the project, I can't find this log anywhere 🤔